src/components/List.js

```diff
@@ -5,14 +5,15 @@ import Contact from './Contact';
 class List extends React.Component {
 
   get items() {
```

Comment:
Rename this to something like getContacts. And remove the 'get' keyword from everywhere. Just have them as simple arrow functions:
`getContacts = () => { doSomething }`